modules/Vtiger/ExecuteFunctions.php

```diff
@@ -95,6 +95,29 @@
 			}
 		}
 		break;
+	case 'getFieldValuesFromMap':
+		$sourceModule = $adb->sql_escape_string(vtlib_purify($_REQUEST['sourceModule']));
+		$inputVal = $adb->sql_escape_string(vtlib_purify($_REQUEST['inputVal']));
+		$fieldname = $adb->sql_escape_string(vtlib_purify($_REQUEST['fieldname']));
+		if (!empty($inputVal)) {
+			$bmapname = $sourceModule . '_FieldInfo';
+			$cbMapid = GlobalVariable::getVariable('BusinessMapping_FieldInfo', cbMap::getMapIdByName($bmapname), $sourceModule, $current_user->id);
+			if (strlen($inputVal) > 0) {
```

Comment:
this condition is unnecessary as you have an `if (!empty($inputVal))` on line 102. If the execution gets here, the length will be bigger than 0 always